examples/grafana-integration/README.md

```diff
@@ -0,0 +1,26 @@
+# Hot R.O.D. - Rides on Demand  - Grafana integration
+
+This is the Hot R.O.D. demo application that consists of the same components as the `examples/hotrod/`, only Grafana and Loki integration is added to this setup, so you can correlate logs and traces in one application.
```

Comment:
It would be useful to explain this in more detail. If someone is new to this whole space, with the main HotROD example we at least have a detailed blog and many videos showing _what to do_ with the example. But here you're assuming substantial familiarity with Grafana and Loki. For example, I think there are some standard Grafana dashboards for Jaeger, could this be included? Or at least a dashboard that shows, ideally, both the HotROD metrics, Loki logs, and traces.

Reply:
+1 for adding a Grafana dashboard to the main `monitoring` directory in the repository, similar to what we have for Jaeger itself:

https://github.com/jaegertracing/jaeger/tree/master/monitoring/jaeger-mixin

Reply:
Yes, I agree, that's exactly what I meant in the PR description. 
I can think of multiple possible ways to extend/transform this:

- I can remove the Loki integration, this way it's a much more simple solution, basically a clean Grafana demo as the frontend to Jaeger -> the setup is more simple, shorter introduction is needed
- I can give additional details to this README to make it easier to understand the setup AND put together an even more detailed blog post, explaining all aspects thoroughly

As for the dashboards:
As far as I know there isn't any official Jaeger dashboard that could be useful for this scenario (if we would like to focus on the tracing aspect of it), since most of them are for monitoring Jaeger itself currently. These can be added, and should be added, just these serve a bit different purpose in my opinion, so having these under `monitoring/` can make sense. 

Although, I've just found [this one](https://grafana.com/grafana/dashboards/12611), which is something similar what could be useful for this setup, but some changes would be needed to tailor to our needs (if it's a welcomed effort, I can work on it).  
Incorporating a dashboard to filter actual logs (and eventually metrics, since those are not yet implemented in this setup) is a greater task, and it might benefit from a more detailed blog post format, but I am happy to work on those as well.
What do you think?

Reply:
I'm all for having a dashboard that serves as the official one, as long as we have people committed to maintain it.

Reply:
Alright, I can open a PR to have one for the logs then.

In that case we would need a setup to validate and use it, so based on these, I'd say the Loki setup should be included as well. 
Although, another option would be to have a dedicated example for vanilla Grafana (for the sake of simplicity), and one for this more advanced one, where Loki and related dashboards are included.

Reply:
as long as the overall setup is simple i.e. `docker-compose up`, I think it makes sense to include Loki integration and have a dashboard that already includes all the relevant panels for HotROD: metrics, logs, and traces.

Reply:
I like the idea. 
I will add Prometheus to this setup too, then start working on a full-fledged HotROD overview dashboard.

Reply:
Just pushed a new config w/ Prometheus integration. 
I will need to look into it more thoroughly, but after taking a quick look at the metrics including errors and the error labels, it's not quite trivial to build useful metrics panels. 

At the moment spotting issues seems easier with Loki, since I can see just filter for `{compose_project="grafana-jaeger"} |= "traceID"` then I can jump to the traces immediately (and this will be even better after https://github.com/jaegertracing/jaeger/pull/2384).  

Is there any dashboard/documentation focusing on the HotROD Prometheus metrics currently to fasten up this part?

Reply:
Added the first version of the HotROD dashboard, reorganized the file structure, extended the README.
Currently it looks like the redis timeouts don't have traceIDs, and the application metrics are not really showing significant error symptoms. 
I had a discussion with @joe-elliott, one of us will check the redis logging, and later I might will be able to open another PR to have new metrics.
Anyway, it looks like the current setup can be considered finished from my perspective. 
Once we have these metrics, I am planning to write a blogpost introducing this new kind of troubleshooting experience.